config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
- root 'homepage#index'
+  resources :clients
+  root 'homepage#index'
+  namespace :admin do
+    root to:'admin#index'
+    resources :admin do
+      member do
+        get 'change_status'
```

Comment:
just to keep things consistent name it update_status

Reply:
changed to update_status.